common/views/components/ResetActiveFilters/ResetActiveFilters.tsx

```diff
@@ -0,0 +1,294 @@
+import React, { FunctionComponent, ReactElement, ReactNode } from 'react';
+import { LinkProps } from '@weco/common/model/link-props';
+import {
+  CatalogueAggregationBucket,
+  CatalogueAggregations,
+} from '@weco/common/model/catalogue';
+import {
+  worksLink,
+  imagesLink,
+  WorksRouteProps,
+} from '../../../services/catalogue/ts_routes';
+import Icon from '../Icon/Icon';
+import Space from '../styled/Space';
+import NextLink from 'next/link';
+import { font, classNames } from '../../../utils/classnames';
+import styled from 'styled-components';
+
+type ResetActiveFilters = {
+  workTypeFilters: CatalogueAggregationBucket[];
+  productionDatesFrom: string | null;
+  productionDatesTo: string | null;
+  worksRouteProps: WorksRouteProps;
+  imagesColor: string | null;
+  workTypeInUrlArray: string[];
+  aggregations?: CatalogueAggregations;
+  resetFilters: LinkProps;
+  enableMoreFilters: boolean;
+  languagesInUrl: string[];
+  subjectsInUrl: string;
+  genresInUrl: string;
+};
+
+const ColorSwatch = styled.span`
+  display: inline-block;
+  width: 12px;
+  height: 12px;
+  background-color: ${({ color }) => color};
+  margin-left: 6px;
+  padding-top: 2px;
+`;
+
+type CancelFilterProps = {
+  text?: string;
+  children?: ReactNode;
```

Comment:
You can just use children here. [like this](https://codesandbox.io/s/mystifying-greider-fx2c3?file=/src/App.tsx:140-208).

Reply:
Can't see what you mean by this ?